### ISSUE TITLE
Model inference example

### ISSUE THREAD

**Issue by Shirobokov-Andrew** [uncbiag/SimpleClick#35]
Hi.
Do you have a simple script with an example of an inference model into which an image and points are fed?

**Comment by qinliuliuqin** [comment]
@Shirobokov-Andrew Yes, I am happy to. Since I am busy with other projects, I will do it once I get time. Maybe in a month. Thank you.

**Comment by reachomk** [comment]
hi, just wanted to follow up on this. We wanted to cite simpleclick as a baseline for a submission we are working on and having this would make it much easier.

**Comment by qinliuliuqin** [comment]
@reachomk Thank you for considering our method as a baseline. I have been thinking about updating this codebase for a year but never get it done. Sorry for that. I will prioritize it and get you updated!

**Comment by reachomk** [comment]
Thanks so much. If I have trained a model using the current codebase, I am hoping you can provide a simple example of how i can run inference on the model, given a single image and an xy point on that image. It looks like the current scripts only support evaluation on datasets like coco without general inference (outside the demo)

The deadline is very soon (< 1 week) so it would be super helpful if you can provide some help.

**Comment by qinliuliuqin** [comment]
@reachomk OK, you have already trained a model based on SimpleClick, and you want to run inference on a single case. This would be easy task, I will do it this Friday and get back to you.

**Comment by reachomk** [comment]
Hi, just wanted to follow up

**Comment by qinliuliuqin** [comment]
@reachomk Hi, I uploaded a simple inference pipeline at here: https://github.com/uncbiag/SimpleClick/blob/v1.0/notebooks/test_simpleclick_inference.py

Let me know if this is what you want.

**Comment by reachomk** [comment]
Thank you, this is super helpful. Really appreciate it!

**Comment by qinliuliuqin** [comment]
@reachomk My pleasure. Feel free to reopen this issue if you have any further questions.
